docs/src/rules/no-magic-numbers.md

```diff
@@ -243,9 +243,9 @@ Examples of **incorrect** code for the `{ "enforceConst": true }` option:
 ```js
 /*eslint no-magic-numbers: ["error", { "enforceConst": true }]*/
 
-var TAX = 0.25;
+const TAX = 0.25;
```

Comment:
```suggestion
let TAX = 0.25;
```
This is an `incorrect` example for `enforceConst: true` option of `no-magic-numbers` which enforces the use of `const` for storing numbers, so here we use `let` to make it `incorrect`.

Reply:
Good catch! I got confused because I thought it was the correct example. Other options have both incorrect and correct eg Should we do the same for the `enforceConst` options as well?

Reply:
Sounds good to me.

Reply:
@Maryyeruva24 pls review the comments and make the necessary revisions.